index.rst

```diff
@@ -177,6 +213,28 @@ If the image is built from a branch that is not ``prod``, and the
 set to a value derived from the branch name.  This prevents building
 standard containers from branches other than ``prod``.
 
+Input Parameters For "Retag" Target
+-----------------------------------
+
+The meaning and defaults for ``input``, ``tag``, and ``supplementary`` differ
```

Comment:
At a glance this section is maybe a bit hard to act on. At its core I think we want a call sample, and then maybe explain the parameters in more detail with a bullet list.

Here's how I might frame this section as a "how-to" for retagging an image to recommended:

```rst
.. make-retag:

Retagging an image to recommended with make retag
-------------------------------------------------

Recommended images are *retagged* from existing images.
You can create a recommended image with the ``make retag`` target:

.. code-block:: sh

   make retag <add params here ...>

The arguments are:

< a bullet list, starting with the parameter name and continuing with a description >
```

Reply:
I took a hybrid approach here: I wanted to keep the parameter definitions and defaults for parallelism with the other targets, but I used the anchor and put the simplest/commonest case (retag a weekly as recommended) inline up here.